CHANGELOG.md

```diff
@@ -3,6 +3,19 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+<a name="0.7.15"></a>
+## [0.7.15](https://github.com/awslabs/jsii/compare/v0.7.14...v0.7.15) (2019-02-27)
+
+
+### Bug Fixes
+
+* **jsii-pacmack:** default to target directory mode ([#363](https://github.com/awslabs/jsii/issues/363)) ([967d917](https://github.com/awslabs/jsii/commit/967d917))
```

Comment:
@RomainMuller keep a closer eye on this during CDK deployments. It changes the structure of the dist/ directory slightly (removes the double "dotnet/dotnet") and there is a chance that our publishers will choke (I expect they don't because I was conscious to use `**/*.nupkg`, but there's always a possibility.

Reply:
Thanks for the heads up. I was aware of this, but I probably wouldn't have paid so much attention nevertheless... 🤷🏻‍♂️